src/jit/gentree.h

```diff
@@ -4240,9 +4240,17 @@ struct GenTreeAddrMode : public GenTreeOp
         return gtOp2;
     }
 
-    unsigned gtScale;  // The scale factor
-    unsigned gtOffset; // The offset to add
+    int Offset()
+    {
+        return static_cast<int>(gtOffset);
+    }
 
+    unsigned gtScale; // The scale factor
+#ifndef LEGACY_BACKEND
+private:
+#endif
+    unsigned gtOffset; // The offset to add
```

Comment:
Eh, like the temporary renaming of `GenTreeIndir::Offset` to `Address` this was a way to ensure that I found all it's usages. Make it private, add `Offset()` and update all uses of `gtOffset` until it builds. It's also used by legacy code and I'd rather not touch that.